Firmware/mmu2_marlin1.cpp

```diff
@@ -27,7 +27,7 @@ bool planner_any_moves() {
 }
 
 float planner_get_machine_position_E_mm(){
-    return current_position[E_AXIS];
+    return st_get_position_mm(E_AXIS);
```

Comment:
we should introduce another function `stepper_get_machine_position_E_mm`

Reply:
Fixed in [9f02bec](https://github.com/prusa3d/Prusa-Firmware/pull/4076/commits/9f02bec7f293092cd5acd811cbe01e622e1f2534)